content/docs/command-reference/update.md

```diff
@@ -53,7 +53,7 @@ $ dvc update --rev master
   [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
   the new data directly to remote storage (the default one unless `-r` is used).
   No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
-  data locally.
+  data locally. This option can't be used with DVC or Git repository imports.
```

Comment:
Thanks @isidentical I copy edited it a little.

Can we also start the paragraph with "update a `.dvc` file created with `dvc import-url` and" instead of the current opening?

Thanks